src/errors/index.ts

```diff
@@ -1,8 +1,9 @@
 export * from './transport';
 export * from './base';
-export * from './not_found';
-export * from './forbidden';
-export * from './unauthorized';
-export * from './internal';
+export * from './notFoundError';
+export * from './forbiddenError';
+export * from './unauthorizedError';
+export * from './internalServerError';
```

Comment:
we use `.error` convention in the file name, ex: `forbidden.error.ts`

Reply:
renamed the files